docs/CONTRIBUTING.md

```diff
@@ -82,6 +108,13 @@ A few tips per log level:
 When running on cloud, the service is running at the `INFO` level. You can activate
 the `DEBUG` level locally with `--debug`
 
+### Linting
```

Comment:
This made me wonder if we should also include running the tests in this checklist?

Reply:
Good call.